### PR TITLE
Sending the Nebai Revelations' proposal

### PR DESCRIPTION
**Pull request by superoptimo**
Sending the proposal of Nebai Revelations' DAOVOTION idea.

## Reasons for making this change
submitting new proposal

### CODE REVIEW

File: proposals/daovotion-nebairevelations-oss-web3.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+### Due diligence
+
+<!-- Please answer the following due diligence questions; it's okay to answer "N/A" if you don't know yet. -->
+
+1. **What related work has already been done in this area?** For decades, OSS have already being sustained by philanthropic funding mostly, but this non-profit approach rarely could effectively sustain consistent efforts in quality assurance, as there aren’t any economical incentives for investors.
+Just recently, **Gitcoin Grants** program were launched with the sole purpose of supporting Public Goods and OSS projects. But as this solution is married with the Quadratic Voting paradigm,  it not only failed to sustain independent OSS projects but also bringed more security issues as this QF model gives perverse incentives for attackers that could steal funds by manipulating the voting system with fake projects.
```
Comment (epicfaace) [inline proposals/daovotion-nebairevelations-oss-web3.md:57]:
```suggestion
Just recently, **Gitcoin Grants** program were launched with the sole purpose of supporting Public Goods and OSS projects. But as this solution is married with the Quadratic Voting paradigm,  it not only failed to sustain independent OSS projects but also brought more security issues as this QF model gives perverse incentives for attackers that could steal funds by manipulating the voting system with fake projects.
```

Comment (epicfaace) [inline proposals/daovotion-nebairevelations-oss-web3.md:57]: Can you explain more in this section why QF doesn't work?

**Review by epicfaace — COMMENTED** [review]
Thanks for your proposal!

File: proposals/daovotion-nebairevelations-oss-web3.md (hunk 1 above)
Comment (superoptimo) [reply proposals/daovotion-nebairevelations-oss-web3.md:57]: Thanks Ashwin. You're very kind in guiding us through this process. We will fix those typos as soon as possible in the next two days.

Comment (superoptimo) [reply proposals/daovotion-nebairevelations-oss-web3.md:57]: Of course. I would be glad to explain the issues with Quadratic Funding. In this section, as you have suggested.

**Review by epicfaace — APPROVED** [review]
Thanks for your submission!

**Pull request merged**